src/main/java/com/yello/server/friend/FriendController.java

```diff
@@ -0,0 +1,44 @@
+package com.yello.server.friend;
+
+import com.yello.server.domain.friend.service.FriendServiceImpl;
+import com.yello.server.domain.vote.dto.response.VoteResponse;
+import com.yello.server.global.common.dto.BaseResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+import static com.yello.server.global.common.SuccessCode.ADD_FRIEND_SUCCESS;
+
+@Tag(name = "01. Friend")
```

Comment:
스웨거에서 01은 Vote가 담당하고 있어서

```suggestion
@Tag(name = "02. Friend")
```

로 고쳐야 할 것 같아요!